examples/extract_data/src/extract_data_login.py

```diff
@@ -131,9 +131,10 @@ async def run():
     
     finally:
         # Clean up
-        client.windows.close(session_id, window.data.window_id)
-        client.sessions.terminate(session_id)
-        print("Session terminated")
+        if client is not None and session_id is not None:
```

Comment:
This throws
![image](https://github.com/user-attachments/assets/027115db-2260-4c50-829a-d3671e12c15a)

You will need to define `session_id` and `window` as None earlier
```
        client = Airtop(api_key=AIRTOP_API_KEY)
        session_id = None
        window = None
```